src/screens/Settings/components/DeleteAccountDialog.tsx

```diff
@@ -0,0 +1,292 @@
+
+    if (!cleanedHandle) {
+      setError(_(msg`Please enter your handle to confirm deletion.`))
+      return
+    }
+
+    if (cleanedHandle.toLowerCase() !== currentAccount.handle.toLowerCase()) {
+      setError(_(msg`Handle doesn't match. Please enter your exact handle.`))
+      return
+    }
+
+    setError('')
+    setIsProcessing(true)
+    const token = confirmCode.replace(/\s/g, '')
+
+    try {
+      // inform chat service of intent to delete account
+      const {success} = await agent.chat.bsky.actor.deleteAccount(undefined, {
+        headers: DM_SERVICE_HEADERS,
+      })
+      if (!success) {
+        throw new Error('Failed to inform chat service of account deletion')
+      }
+      await agent.com.atproto.server.deleteAccount({
+        did: currentAccount.did,
+        password,
+        token,
+      })
+      control.close(() => {
+        Toast.show(_(msg`Your account has been deleted`))
+        resetToTab('HomeTab')
+        removeAccount(currentAccount)
+      })
+    } catch (e: any) {
+      setError(cleanError(e))
+    }
+    setIsProcessing(false)
+  }
+
+  const title =
+    stage === Stages.SendEmail
+      ? currentAccount
+        ? _(msg`Delete account "${sanitizeHandle(currentAccount?.handle)}"`)
+        : _(msg`Delete account`)
+      : _(msg`Confirm deletion`)
+
+  return (
+    <Dialog.ScrollableInner
+      label={_(msg`Delete account dialog`)}
+      style={web({maxWidth: 450})}>
+      <View style={[a.gap_xl]}>
+        <View style={[a.gap_sm]}>
+          <Text style={[a.font_heavy, a.text_2xl]}>{title}</Text>
+
+          {error ? (
+            <View style={[a.rounded_sm, a.overflow_hidden]}>
+              <ErrorMessage message={error} />
+            </View>
+          ) : null}
+
+          <Text style={[a.text_md, a.leading_snug]}>
+            {stage === Stages.SendEmail ? (
+              <Trans>
+                For security reasons, we'll need to send a confirmation code to
+                your email address.
+              </Trans>
+            ) : (
+              <Trans>
+                Check your inbox for an email with the confirmation code to
+                enter below:
+              </Trans>
+            )}
+          </Text>
+        </View>
+
+        {stage === Stages.SendEmail && (
+          <Admonition>
+            <Trans>
+              You can also{' '}
+              <RNText
+                style={[{color: t.palette.primary_500}, web(a.underline)]}
+                onPress={() =>
+                  control.close(() => deactivateDialogControl.open())
+                }>
+                temporarily deactivate
+              </RNText>{' '}
+              your account instead, and reactivate it at any time.
+            </Trans>
+          </Admonition>
+        )}
+
+        {stage === Stages.DeleteAccount && (
+          <View style={[a.gap_md]}>
+            <View>
+              <TextField.LabelText>
+                <Trans>Confirmation Code</Trans>
```

Comment:
```suggestion
                <Trans>Confirmation code</Trans>
```
We currently have quite a few instances where this string is used in sentence case, would that work here?

![IMG_0959](https://github.com/user-attachments/assets/d7bd996a-0408-4b87-a075-ab81209fb958)